### ISSUE TITLE
Distribution

### ISSUE THREAD

**Issue by matt-hu** [SartoxOnlyGNU/audacium#28]
Ok, so now we have binary builds, we need to start thinking about getting our software out.

**Current places**
- AUR - git build

**Places we need to submit to**
- AUR (binary builds, *much* quicker to install)
- WinGet - need an installer first

**People we need to contact**
- [ ] Debian (?)
- [ ] Raspberry Pi
- [ ] Ubuntu (?)

Please let me know if I've missed any, I'll edit this as/when we start getting the software included.

**Comment by AnErrupTion** [comment]
I doubt Debian and Ubuntu will accept our request right now as they have an outdated wxWidgets version in their repositories, and since Raspbian uses Debian as their base they might also have an outdated version.

**Comment by matt-hu** [comment]
Ah ok, I wasn't aware of that. I'm guessing there isn't any way to bundle wxWidgets with audacium?

**Comment by AnErrupTion** [comment]
It's already bundled, but I think package managers don't like that.

**Comment by pcnerd19** [comment]
I'm not sure if ElementaryOS had their own repos, but I know Manjaro does so that might be another big distro to look at (if you want to go past the AUR). It's not my personal favorite, but I know a lot of people use Snap or Flatpak so that could also be a good place to get it for wide and easy availability.

**Comment by AnErrupTion** [comment]
I hate snaps and flatpaks but we'll see if we can integrate Audacium into them. Also, no WinGet, it requires signing a CLA.

**Comment by AnErrupTion** [comment]
@matt-hu Any progress?

**Comment by pjalsGit** [comment]
Fedora/RedHat would be nice, even if it would just be [COPR](https://copr.fedorainfracloud.org/coprs)

**Comment by AnErrupTion** [comment]
@pjalsGit Agreed, but we have yet to know the status of the current distribution.

**Comment by ewanm89** [comment]
> Also, no WinGet, it requires signing a CLA.

Winget CLA is only for the manifest YAML so it know where to get the installer from and what version it is.

**Comment by AnErrupTion** [comment]
Ah, alright. I'll check it out then!

**Comment by jt15s** [comment]
Maybe getting Audacium on the Microsoft Store would be good too?

**Comment by AnErrupTion** [comment]
@jt15s Yup, I'll look into that too!

**Comment by AnErrupTion** [comment]
Also, someone updated Audacium to 1.0.0 in WinGet: https://github.com/microsoft/winget-pkgs/pull/37770

**Comment by ewanm89** [comment]
Hehe, someone must have posted that not long after my message about the CLA
